requirements/base.txt

```diff
@@ -21,7 +21,7 @@ django-js-asset==2.0.0
     # via django-mptt
 django-mptt==0.13.4
     # via -r requirements/base.in
-django-sekizai==3.0.1
+django-sekizai==4.0.0
```

Comment:
Support for Django 2 and 3.1 dropped.
https://github.com/django-cms/django-sekizai/releases/tag/4.0.0